### DISCUSSION TITLE
deformable sprite console error

### DISCUSSION THREAD

**Discussion by LePiromano** [FakeFishGames/Barotrauma#14237] [Bug reports]
### Disclaimers

- [X] I have searched the issue tracker to check if the issue has already been reported.
- [ ] My issue happened while using mods.

### What happened?

![image](https://github.com/FakeFishGames/Barotrauma/assets/55637865/9a884a0e-c49a-4316-88cd-a58d47dc2493)
99.9% sure hammerhead matriarchs and molochs cause it, i've only seen it twice and met either of those

### Reproduction steps

load in a killing hammerheads mission or moloch mission, see if it happens

### Bug prevalence

Happens every now and then

### Single player or multiplayer?

Single player

### Version

v1.6.100.0 (unstable)

### Which operating system did you encounter this bug on?

Windows

**Comment by Regalis11** [comment]
Fixed in our private repo https://github.com/FakeFishGames/Barotrauma-development/pull/5532, fix coming up in a future build!

**Comment by NilanthAnimosus** [comment]
Fix tested and merged with the codebase, should be in a future version!
